src/java.base/share/classes/java/util/BitSetReadOps.java

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 1995, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Drop 1995?

Reply:
I think 1995 should remain as the base of this class emanates from old code?